web/sessions.go

```diff
@@ -216,6 +238,21 @@ func (mss *mongoDBSessionStore) Get(ctx context.Context, id string) (*Session, e
 	return s, nil
 }
 
+func (mss *mongoDBSessionStore) HasSessionWithToken(ctx context.Context, token string) (bool, error) {
+	ctx, span := trace.StartSpan(ctx, "MongoDBSessionStore::Get")
+	defer span.End()
+
+	count, err := mss.collection.CountDocuments(ctx, bson.M{accessTokenFieldPath: token}, options.Count().SetLimit(1))
+	if err != nil {
+		if errors.Is(err, mongo.ErrNoDocuments) {
+			return false, errNoSession
+		}
```

Comment:
Unless I'm misunderstanding what `mongo.ErrNoDocuments` means, should we consider this an expected case and avoid returning the error? The `error` level log up top seems a little aggressive for the normal `count == 0` case

Reply:
oh yeah this doenst make much sense with the new function name. i actually dont think i need this check now that i moved it from FindOne. count return this error. will remove!